section2/reflection.md

```diff
@@ -1,15 +1,59 @@
+    taught in the growth mindset lesson. The act of repetition allowed me to remember how and when  
+    to run my files in node. Also, the vague instructions gave me the opportunity to use what I've
+    learned thus far to complete each task. Overall, Section 2 did challenge me and I felt proud
+    that I realized the opportunity to tap in to my growth mindset while outside of my comfort-zone.
 
-3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
-5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+   The conditional statements in JS are **if, else, else if and switch-case statements**.  
+   They are used for checking a specific condition and executing the code based on the specified condition.  
+
+   **Daily Life Example:**   
+   If I chose to write with a pencil, I can erase my work if I make a mistake.  
+   If I chose to write with a pen, I can't erase my work if I make a mistake.  
+
+   **Web-app Example:**
+   On BeatStars, users can listen to beats made by indie producers. A simple example of where a conditional statement may be used is when a user clicks on a beat's mp3 player. It may say something like: "If user clicks the play button, change the play icon to the pause icon ".
+
+5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.  
+
+   You add multiple conditions to an `if` statement by combining it with `else` and `else if` statements.
+   The program will check each condition and run each specified task based on the rules that are defined by the developer.
+
+6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?  
 
-6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+  I can use the logical-and operator (`&&`), to check that that both conditions are met.
+  I can use the logical-or operator (`||`) to check that one of 2 conditions are met.
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions?  
+   Why is it not recommended to use an `if` statement with an assignment like "`x=y`" as a condition?
```

Comment:
If what you are asking is about the `==` as opposed to `=` (please correct me if I read your question incorrectly) - you'll always get a truthy value, therefore are setting yourself up to get a false positive at some point.

I think of it as: 
- `=` _instructs_ JS to assign a value to a variable 
- `==` ASKS JS if two values have the same value